### PR TITLE
[13.0][MIG] stock_orderpoint_manual_procurement

### PR DESCRIPTION
**Pull request by JoanSForgeFlow**
stock_orderpoint_manual_procurement migration to v13

cc ~ @ForgeFlow @LoisRForgeFlow

### CODE REVIEW

**Review by AdriaGForgeFlow — CHANGES_REQUESTED** [review]
For some reason, when I execute manually the Request Procurement action, in the RFQ the quantity is twice the procure recommendation that appeared in the wizard.

manual_orderpoint

**Comment by JoanSForgeFlow** [comment]
comments attended

**Review by AdriaGForgeFlow — APPROVED** [review]

**Comment by AdriaGForgeFlow** [comment]
> comments attended

Thanks! :+1:

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

File: stock_orderpoint_manual_procurement/wizards/make_procurement_orderpoint.py [hunk 1]
```diff
@@ -77,15 +76,20 @@ def make_procurement(self):
...
+                        item.uom_id,
+                        item.orderpoint_id.location_id,
+                        item.orderpoint_id.name,
+                        item.orderpoint_id.name,
+                        item.orderpoint_id.company_id,
+                        values,
+                    )
                 )
+                pg_obj.run(procurements)
+                procurements.clear()
```
Comment (LoisRForgeFlow) [inline stock_orderpoint_manual_procurement/wizards/make_procurement_orderpoint.py]: This is inefficient. Odoo added the `Procurement` namedtuple exactly for this, to be able to do several procurements with one call to `run` method. Please, have a look [at this](https://github.com/OCA/manufacture/blob/13.0/mrp_multi_level/wizards/mrp_inventory_procure.py#L89) and follow the same approach: first construct a list of procurements and after that call `run` method just once.

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
Please see my comment below

**Review by LoisRForgeFlow — APPROVED** [review]
LG now, thanks!

**Pull request merged**